src/cbmc_starter_kit/version.py

```diff
@@ -21,3 +21,15 @@ def copy_with_version(src, dst):
         data = srcfile.read()
     with open(dst, 'w', encoding='utf-8') as dstfile:
         dstfile.write(data.replace(REPLACE_TARGET, version()))
+
+def update_existing_version_in_workflow_file(workflow):
+    with open(workflow, encoding='utf-8') as srcfile:
```

Comment:
why is `encoding` needed?

Reply:
Removed. Indeed extraneous for this case.